firmware/Components.cpp

```diff
@@ -62,6 +62,8 @@ void Seaking::arm()
 {
   disable();
   delay(500);
+  disable();
+  delay(500);
```

Comment:
Why do we disable twice here?  We could just wait 1s after the first disable, right?

Or is this a typo?